content/en/docs/concepts/architecture/nodes.md

```diff
@@ -412,6 +413,20 @@ thus not activating the graceful node shutdown functionality.
 To activate the feature, the two kubelet config settings should be configured appropriately and
 set to non-zero values.
 
+Once systemd detects or notifies node shutdown, the kubelet sets a `NotReady` condition on
+the Node, with the `reason` set to `"node is shutting down"`. The kube-scheduler honors this condition
+and does not schedule any Pods onto the affected node; other third-party schedulers are
+expected to follow the same logic. This means that new Pods won't be scheduled onto that node
+and therefore none will start.
+
+The kubelet **also** rejects Pods during the `PodAdmission` phase if an ongoing
+node shutdown has been detected, so that even Pods with a
+{{< glossary_tooltip text="toleration" term_id="toleration" >}} for
+`node.kubernetes.io/not-ready:NoSchedule` do not start there.
```

Comment:
This is good to know.
How about static Pods? If I or an automation tool happened to add a new manifest
or changing an existing one, without noticing that the node is being shut down,
what will happen?

Reply:
the admissions will not allow anything: https://github.com/kubernetes/kubernetes/blob/53636bc7806d6e4fa1e9a4824d04779d0c4cde39/pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go#L146-L157